setup.py

```diff
@@ -1,32 +1,37 @@
+# Import required packages
 from setuptools import setup
 
-with open("README.md", "r") as f:
-    long_description = f.read()
+# Open the README file and read the contents into a variable
+with open("README.md", "r") as readme_file:
+    long_description = readme_file.read()
 
-test_reqs = [
-    'mock==2.0.0'
+# Define lists of test and documentation requirements
+test_requirements = [
+    "mock==2.0.0"
 ]
 
-doc_reqs = [
-    'Sphinx>=1.5.2'
+doc_requirements = [
+    "Sphinx>=1.5.2"
 ]
 
-extra_reqs = {
-    'doc': doc_reqs,
-    'test': test_reqs
+# Define a dictionary of extra requirements
+extra_requirements = {
+    "doc": doc_requirements,
+    "test": test_requirements
 }
 
+# Configure the setup function with required and optional parameters
 setup(
-    name='spotipy',
-    version='2.23.0',
-    description='A light weight Python library for the Spotify Web API',
+    name="spotipy",
```

Comment:
We definitely need consistency with single and double quotes, but this should be managed on the whole project at the linter level. Feel free to open a PR that does only that.